tasks/task_116_com2sense_commonsense_reasoning.json

```diff
@@ -0,0 +1,12051 @@
+{
+    "Definition": "You will be given one sentence about an everyday event. You need to decide whether the sentence is true or false, based on your commonsense. For true sentences output 'True' and for false sentences output 'False'.",
```

Comment:
I am not sure what it means for a sentence to be "true" or "false". 
I would suggest changing the phrasing of the instruction, to something along the line of, "you need to decide whether provided story in the following sentence is a plausible scenario and it matches your commonsense".

Reply:
Similarly, instead of saying "For true sentences output 'True'" (which sounds like a recursive statement), say, "if the given event is a plausible one, output 'True'" and so on.